src/interfaces/coral_web/src/components/Welcome.tsx

```diff
@@ -0,0 +1,37 @@
+import { Transition } from '@headlessui/react';
+import React from 'react';
+
+import { BotAvatar } from '@/components/Avatar';
+import { Text } from '@/components/Shared';
+import { BotState } from '@/types/message';
+
+type Props = {
+  show: boolean;
```

Comment:
Should we  receive props here, so that it can be reused for the <Welcome /> of chatbot assistants?

something like:

```ts
type Props = {
  isBaseAgent: boolean,
  name: string,
  description: string
}
```

Reply:
yea i think we would need that to make it work with assistants, these are just ported over from coral! i can make that change in a follow up :)